pkg/controller/account/account_controller.go

```diff
@@ -813,7 +867,7 @@ func formatAccountEmail(name string) string {
 }
 
 // TotalAwsAccounts returns the total number of aws accounts in the aws org
-func TotalAwsAccounts(client awsclient.Client) (int, error) {
+func TotalAwsAccounts(reqLogger logr.Logger, client awsclient.Client) (int, error) {
```

Comment:
this is being used in localmetrics line 101 and in account_controller_test line 114, which has not been updated with the reqLogger. Those 2 instances needs to incorporate in the reqLogger as well